app/controllers/api/v1/service_offerings_controller.rb

```diff
@@ -3,6 +3,48 @@ module V1
     class ServiceOfferingsController < ApplicationController
       include Api::V1::Mixins::IndexMixin
       include Api::V1::Mixins::ShowMixin
+      include Api::V1::Mixins::SourcesApiMixin
+
+      def order
+        service_offering = model.find(params_for_order[:service_offering_id].to_i)
```

Comment:
@agrare do we always include `tenant_id => XY` into a query?

Reply:
I believe acts_as_tenant automatically adds the current tenant to the query but we should confirm

Reply:
probably yes, I don't have tenancy disabled on localhost

Reply:
Should be easy to confirm by checking the rails log for the query it executes

Reply:
```
SELECT  "service_offerings".* FROM "service_offerings" WHERE "servic
e_offerings"."tenant_id" = $1 AND "service_offerings"."id" = $2 LIMIT $3  [["tenant_id", 1], ["id", 3], ["LIMIT", 1]]
```

Reply:
awesome, I wasn't sure if find was covered

Reply:
:heart: perfect, lgtm